website/cue/reference/components/sinks.cue

```diff
@@ -163,7 +163,39 @@ components: sinks: [Name=string]: {
 								required:    true
 								type: string: {
 									examples: features.send.encoding.codec.enum
-									syntax:   "literal"
+									let batched = features.send.encoding.codec.batched
+									enum: {
+										for codec in features.send.encoding.codec.enum {
```

Comment:
I imagine I could refactor this if I had more cue-fu, but this is the best I could come up with.